voice/bxml/callbacks/conferenceCreated.md

```diff
@@ -1,19 +1,28 @@
 {% method %}
 ## Conference Created Event
 The Conference Created event is fired whenever a new [conference](../verbs/conference.md) that specified a callbackUrl is created.
+
+`<PlayAudio>` and `<SpeakSentence>` verbs can be used in BXML returned by this callback to play audio in the conference, which will be heard by members that enter the conference while still playing.
+
 ### Expected response
+
 ```http
-HTTP/1.1 204
+HTTP/1.1 200
+Content-Type: application/xml; charset=utf-8
+
+<Response>
+    <!-- <PlayAudio> or <SpeakSentence> BXML verbs to play in the conference -->
+</Response>
```

Comment:
Might be nice to have examples of no response and a response

Reply:
Added back the 204 response example.